documentation/certificates.md

```diff
@@ -2,24 +2,44 @@ Streisand PKI
 =============
 
 Streisand includes a role responsible for PKI certificate generation,
-separated into two distinct tasks:
+separated into three heirarchical tasks:
 
- - CA/Server
- - Client(s)
+```
+CA/Server
+   \__ CA Certificate
+     \__ Server certificate
+    \__ Client (1..n)
+      \__ Client PKCS (1..n)
+ 
+```
 
-The first will generate a certificate authority, and a server 
+CA/Server
+---------
+Guarded by `generate_ca_server: yes`
+
+Generate a certificate authority, and a server 
 certificate signed by the newly minted certificate authority.
 
-The latter task is responsible for generating client side certificates
-(using the same CA) used for authenticating connecting VPN clients.
+Client
+------
+Guarded by `generate_client: yes`
+
+Generate client side certificates using the same CA used for authenticating
```

Comment:
I think this sentence is a little confusing because it's mildly self-referential. The CA authenticates VPN clients using the client side certificates that are generated by the CA to authenticate VPN clients :-)

How about:

"Generate client certificates using the CA from `generate_ca_server: yes`. These client certificates can be used by VPN clients to authenticate with VPN services on the server." ?